### DISCUSSION TITLE
Setting branch names

### DISCUSSION THREAD

**Discussion by Balaika** [ropensci/targets#738] [Help]
Hi,

Thanks for the great targets package.
I wanted to know whether it is possible to choose branch names with dynamic branching ?
 
I mean having names such as y_model1, y_model2 rather than y_4096b6a8, y_61ced14d would help a lot.

**Comment by wlandau** [comment]
Sorry, I do not have plans to implement custom names for dynamic branches. It turns out to not be a good fit with how dynamic branching works. In the general case, we do not know what the names should be because we do not know in advance what targets will be defined or how many there will be. Picking names requires prior knowledge of the outcome. See https://github.com/ropensci/targets/discussions/668 for a longer discussion.

**Reply by Balaika** [reply]
Ok I understand. I see I could use static branching. In general I tend to prefer the dynamic branching syntax even when I can use static branching because I feel the code is more readable.
